recipes/mling_pl/README.md

```diff
@@ -29,28 +29,32 @@ Prepare a file with the list of audio files in this format
 
 #### Step 4:
 
-Run inference using the following command from flashlight build directory
+Run inference using the following command from flashlight build directory:
 
 ```
 bin/asr/fl_asr_test \
-    --test <audio_file_list> \
-    --am <path_to_model_checkpoint.bin> \
-    --arch <path_to_model_arch.so> \
-    --tokens <path_to_tokens_file/tokens-all.lst> \
-    --datadir ''  \
-    --emission_dir ''  \
+    --test=<audio_file_list> \
+    --am=<path_to_model_checkpoint.bin> \
+    --arch=<path_to_model_arch.so> \
+    --tokens=<path_to_tokens_file/tokens-all.lst> \
+    --lexicon=lexicon.txt \
+    --datadir=''  \
+    --emission_dir=''  \
     --show
 ```
 
 To compile `*.cpp` architectures into `*.so` use cmake/make command in flashlight and provide `-DFL_PLUGIN_MODULE_SRC_PATH=path/to/*.cpp` flag.
 
+A lexicon file is required for inference, but because we use greedy decoding, the lexicon isn't actually used. You can create a dummy lexicon using this command: `echo 'a a |' > lexicon.txt`
+
+A Colab notebook with an example of using the model can be found [here](https://colab.research.google.com/drive/1xObuBVyZcPj8JXQm-gWa3e5WG0wgDreZ).
```

Comment:
Could you add the notebook to the same folder so that other users can also modify the notebook if needed and keep track of changes.
 
See https://github.com/flashlight/flashlight/tree/main/flashlight/app/asr/tutorial for an example.

Reply:
Done.